specification/purview/data-plane/Azure.Analytics.Purview.Catalog/preview/2021-05-01-preview/purviewcatalog.json

```diff
@@ -2235,7 +2235,7 @@
           "Lineage"
         ],
         "description": "Get lineage info of the entity specified by GUID.",
-        "operationId": "GetLineageGraph",
+        "operationId": "Lineage_GetLineageGraph",
```

Comment:
@ellismg I'd like to get your two cents precisely here

Reply:
@yifan-zhou922 why this change? This is breaking SDKs, from what feedback do you want to do this?

Reply:
To explain the motivation behind the change: The removal of the Lineage prefix was so that we would not create a separate lineage client with just two operations.

Reply:
https://review.docs.microsoft.com/en-us/rest/api/purview/catalogdataplane/getlineagegraph?branch=catalog This is the generated document which looks strange because lineage is not displayed at the same level as Entity, Glossary, etc. In the future, we might add new APIs under lineage category. So we want to add lineage prefix.